styles.css/main.css

```diff
@@ -79,10 +79,7 @@ nav ul {
   list-style-type: none;
   text-decoration: none;
   display: flex;
-  align-items: center;
   justify-content: center;
-  flex-direction: column;
-  align-items: flex-start;
 }
```

Comment:
You placed a number of properties that were intended for narrow screens inside the main rules that apply at desktop sizes.  You need to use media queries to have certain rules/properties only apply to narrower screens